scripts/test-pipelines.sh

```diff
@@ -59,7 +59,7 @@ test $test_dir:
   interruptible: true
   script:
     - >
-        nix-shell -I nixpkgs=./pkgs.nix --packages nodejs --run '
+        nix-shell -I nixpkgs=./pkgs.nix --packages nodejs iproute2 utillinux nftables iptables --run '
```

Comment:
Since we are using `iptables-legacy`, this can just be changed to `iptables-legacy`.